src/js/jsx/sections/style/Stroke.jsx

```diff
@@ -232,94 +244,49 @@ define(function (require, exports, module) {
-                                                {strings.STYLE.STROKE.ALIGNMENT}
-                                            </Label>
-                                            <StrokeAlignment
-                                                {...this.props}
-                                                layers={this.state.layers}
-                                                alignments={stroke.alignments}/>
-                                        </div>
-                                    </div>
-                                </ColorInput>
-                                <Gutter />
-                                <ToggleButton
-                                    title={strings.TOOLTIPS.TOGGLE_STROKE}
-                                    name="toggleStrokeEnabled"
-                                    buttonType={"layer-not-visible"}
-                                    selected={stroke.enabledFlags}
-                                    selectedButtonType={"layer-visible"}
-                                    onClick={!this.props.disabled ? this._toggleStrokeEnabled : _.noop}
-                                    size="column-2"
-                                />
-                                <Gutter />
-                            </div>
-                        </div>
+                <div className="formline">
+                    <div className="control-group__vertical">
+                        <ColorInput
+                            id={colorInputID}
+                            className="stroke"
+                            context={collection.pluck(this.state.layers, "id")}
+                            title={strings.TOOLTIPS.SET_STROKE_COLOR}
+                            editable={!this.props.disabled}
+                            defaultValue={stroke.colors}
+                            onChange={this._colorChanged}
+                            onFocus={this.props.onFocus}
+                            onColorChange={this._opaqueColorChanged}
+                            onAlphaChange={this._alphaChanged}
+                            onClick={!this.props.disabled ? this._toggleColorPicker : _.noop}
+                            swatchOverlay={strokeOverlay} />
+                    </div>
+                    <div className="control-group__vertical control-group__no-label column-9">
+                        <NumberInput
+                            value={stroke.widths}
+                            onChange={this._widthChanged}
+                            onFocus={this.props.onFocus}
+                            min={0}
+                            step={1}
+                            bigstep={5}
+                            disabled={this.props.disabled}
+                            size="column-6" />
+                    </div>
+                    <div className={strokeClasses}>
+                        <StrokeAlignment
+                            {...this.props}
```

Comment:
Can you expand the props by only passing what are required by StrokeAlignment component? This will make clear the data flow between the parent and child component.